calypso_ros:
  ros__parameters:
    logging: true
    usb_port: "/dev/ttyUSB0"
    baud_rate: 115200
    log_directory: "Default"
